src/class_singleCleaner.py

```diff
@@ -73,7 +74,7 @@ def run(self):
             # If we are running as a daemon then we are going to fill up the UI
             # queue which will never be handled by a UI. We should clear it to
             # save memory.
-            # FIXME redundant?
+            # ..fixme:: redundant?
```

Comment:
Again, where did you find such reST directive?

Reply:
I didn't understand what you're trying to say?

Reply:
What are you trying to achieve by this replacement? Is it for [doc](https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.class_singleCleaner.html)? If so, you should reference the Sphinx [reST syntax](https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/index.html). I cannot found such directive there. It has `..todo:`, but the result [is not very helpful](https://pybitmessage-test.readthedocs.io/en/latest/contribute.dir/develop.dir/todo.html). Also it seems Sphinx [doesn't get them](https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/network/udp.py#L28) from comments, only from docstrings: https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.network.udp.html.

Reply:
Look: https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.class_singleCleaner.html
I merged this branch into my `testing`. And there is your hypotetic `.. fixed:` directive result?

Reply:
> What are you trying to achieve by this replacement? Is it for [doc](https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.class_singleCleaner.html)? If so, you should reference the Sphinx [reST syntax](https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/index.html). I cannot found such directive there. It has `..todo:`, but the result [is not very helpful](https://pybitmessage-test.readthedocs.io/en/latest/contribute.dir/develop.dir/todo.html). Also it seems Sphinx [doesn't get them](https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/network/udp.py#L28) from comments, only from docstrings: https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.network.udp.html.

Pylint warned me with this FIXME:: **W:135, 0: FIXME redundant? (fixme)**

Reply:
> What are you trying to achieve by this replacement? Is it for [doc](https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.class_singleCleaner.html)? If so, you should reference the Sphinx [reST syntax](https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/index.html). I cannot found such directive there. It has `..todo:`, but the result [is not very helpful](https://pybitmessage-test.readthedocs.io/en/latest/contribute.dir/develop.dir/todo.html). Also it seems Sphinx [doesn't get them](https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/network/udp.py#L28) from comments, only from docstrings: https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.network.udp.html.

Pylint warned me with this FIXME:: **W:135, 0: FIXME redundant? (fixme)**

Reply:
> What are you trying to achieve by this replacement? Is it for [doc](https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.class_singleCleaner.html)? If so, you should reference the Sphinx [reST syntax](https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/index.html). I cannot found such directive there. It has `..todo:`, but the result [is not very helpful](https://pybitmessage-test.readthedocs.io/en/latest/contribute.dir/develop.dir/todo.html). Also it seems Sphinx [doesn't get them](https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/network/udp.py#L28) from comments, only from docstrings: https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.network.udp.html.

Pylint warned me with this FIXME:: **W: FIXME redundant? (fixme)**

Reply:
Just let it warn